components/openindiana/illumos-gate/Makefile

```diff
@@ -106,7 +105,7 @@ $(BUILD_DIR)/$(MACH)/.built: $(SOURCE_DIR)/.patched $(BUILD_DIR)/runtime-perl.p5
 	  echo export PKGVERS_BRANCH=$(ONNV_BUILDNUM); \
 	  echo export BOOTBANNER1=\"$(DISTRIBUTION_NAME) $(DISTRIBUTION_VERSION) Version ^v ^w-bit\") > \
 	  illumos.sh && \
-	  time $(ENV) -i ./usr/src/tools/scripts/nightly illumos.sh
+	  time $(ENV) ./usr/src/tools/scripts/nightly illumos.sh -V $(shell git log -1 --format=illumos-%h)
```

Comment:
Why removal of `-i` here?

Reply:
There is already an '-i' in there apparently. It's visible in the logs.

Reply:
Ah, you are right.  `ENV` got changed in January to contain `-i`. :-)